infra-as-code/bicep/CRML/subscriptionAlias/README.md

```diff
@@ -41,14 +41,13 @@ In this example, the Subscription is created upon an EA Account through a tenant
 **NOTE: As there is some PowerShell code within the CLI, there is a requirement to execute the deployments in a cross-platform terminal which has PowerShell installed.**
```

Comment:
```suggestion
```

Sorry missed one, this need to be removed

Reply:
done